app/src/components/messageParser.js

```diff
@@ -26,9 +26,12 @@ const messageParser = {
    *  @returns {object[]} Returns an array of JSON Object for HALPAS logstash
    */
   parseMany: async (authorizedParty, obj) => {
-    return await Promise.all(
-      obj.map(entry => messageParser.parse(authorizedParty, entry))
-    );
+    const txid = uuid.v4();
+    const ts = moment.utc().valueOf();
+    obj.forEach((item, idx) => {
+      item.transaction = {batch: {id: txid, size: obj.length, itemId:idx+1, timestamp:ts}};
```

Comment:
Inconsistent spacing after colon. Make sure to be using the code auto-formatter whenever possible.

Reply:
lint:fix doesn't pick this up.  not all devs use the same ide and can't depend on a particular auto-formatter.  it's not an enforceable format if you're depending on your ide.

fixed